spec/inputs/country_input_spec.rb

```diff
@@ -56,7 +56,7 @@
       priority_countries = ["Foo", "Bah"]
       semantic_form_for(@new_post) do |builder|
         builder.stub(:country_select).and_return(Formtastic::Util.html_safe("<select><option>...</option></select>"))
-        builder.should_receive(:country_select).with(:country, priority_countries, {}, {:id => "post_country", :required => false, :autofocus => false}).and_return(Formtastic::Util.html_safe("<select><option>...</option></select>"))
+        builder.should_receive(:country_select).with(:country, priority_countries, {}, {:id => "post_country", :required => false, :autofocus => false, :readonly => false}).and_return(Formtastic::Util.html_safe("<select><option>...</option></select>"))
```

Comment:
Why did these existing tests need to change? Was it a quick hack to get the tests to pass, or is it because the `:readonly` option will always be passed from now on?